<launch>

  <node pkg="giskard_examples" type="pr2_segfollower" name="pr2_segfollower" output="screen">
    <rosparam param="joint_names"
      file="$(find giskard_examples)/config/pr2_left_upper_body.yaml" />
    <param name="controller_description" 
      textfile="$(find giskard_examples)/controller_specs/pr2_seg_follow_control.yaml" />
    <remap from="~joint_states" to="/joint_states" />
  </node>
 
  <node pkg="giskard_examples" type="pr2_point2point" name="pr2_point2point" output="screen">
    <rosparam param="joint_names"
      file="$(find giskard_examples)/config/pr2_left_upper_body.yaml" />
    <param name="controller_description" 
      textfile="$(find giskard_examples)/controller_specs/pr2_point2point_control.yaml" />
    <remap from="~joint_states" to="/joint_states" />
  </node>
 
</launch>
